src/util/file_system.cpp

```diff
@@ -146,7 +146,7 @@ std::string normalize(const std::string& filename)
       if (path_stack.empty()) {
 
         log_warning << "Invalid '..' in path '" << filename << "'" << std::endl;
-        // push it into the result path so that the user sees his error...
+        // Push it into the result path so that the user sees their error...
```

Comment:
I believe the original comment may have typo-ed "this", so here it should probably be "this", instead of "their"?